deploy/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
Je hebt nu zowel de files `wait-for-it.sh` en `docker-wait.sh`. Beide proberen hetzelfde te doen. Ik zou zelf voor `wait-for-it.sh` gaan omdat die door Docker docs wordt aangeraden, en dan je project in lijn is met de andere projecten. 

`docker-wait.sh` kan dan weg.